RELEASE.md

```diff
@@ -1,3 +1,25 @@
+# Release 1.3.0
+
+## New features
+
+- Dockerfile optimisation (https://github.com/marqo-ai/marqo/pull/569/files). A pre-built Marqo base image results in reduced image layers and increased build speed.
```

Comment:
We can also add the benefit of `increased build speed.`: `...allowing a better docker pull and development experience` 
(as in community developers are the target of this benefit)